docs/dev-guides/tutorials/why-we-are-doing-it.md

```diff
@@ -0,0 +1,34 @@
+# Why We Are Doing It
```

Comment:
We need to be more specific with titles for people landing from outside nav. Avoid using nonspecific language like "it" generally. This should be something like:

- "Reasons to use our APIs"
- "Advantages of using SDKs and APIs"
- "When to use our API"